pronotepy/ent/generic_func.py

```diff
@@ -179,7 +221,7 @@ def _open_ent_ng_edu(
     params = {"providerId": f"{domain}/auth/saml/metadata/idp.xml"}
 
     response = session.get(ent_login_page, params=params, headers=HEADERS)
-    response = educonnect(response.url, session, username, password)
+    response = _educonnect(session, username, password, response.url, exceptions=False)
```

Comment:
I would just catch the exception, but this works too.

Reply:
No because I need in this case to don't catch the exception to use an other Auth method if it doesn't work

Reply:
I still don't get it, but fine lol